src/Parser/Core/Modules/Features/WarningDisplay.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import WarningIcon from 'Icons/Warning';
+
+class WarningDisplay extends Analyzer {
+
+  render() {
+    const boss = this.owner.boss;
+    if (boss && boss.fight.displayAccuracyWarning) {
+      return (
+        <div>
+          <div style={{ padding: 0 }}>
+            <div className="flex-sub content-middle" style={{ margin: '10px 30px 10px 30px', color: '#FC9F06', opacity: 0.8 }}>
+                <div style={{ fontSize: '4em', lineHeight: 1, marginRight: 20 }}>
+                  <WarningIcon />
+                </div>
+                <div>
+                  <h5>
```

Comment:
You shouldn't use `h5` for styling; it's considered unused and its styling might change in the future. If you want to give the any specific styling you can use the `style` attribute on the `div`.